packages/vhd-lib/src/vhd.js

```diff
@@ -178,19 +178,25 @@ export default class Vhd {
   }
 
   // return the first sector (bitmap) of a block
-  _getBatEntry(block) {
-    const i = block * 4
+  _getBatEntry(blockId) {
+    const i = blockId * 4
```

Comment:
What's this change?

Reply:
it makes it clearer that it's not a block but just an Id. I had trouble reading, and just though of making a small improvement for the other people reading after me.

Reply:
I'll checkout these changes directly into `master`.